cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1196,6 +1196,34 @@ declare namespace Deno {
     bytesSentData: number;
     bytesReceived: number;
   }
+
+  export interface RequestEvent {
+    readonly request: Request;
+    respondWith(r: Response | Promise<Response>): void;
+  }
+
+  export interface RequestIter extends AsyncIterator<RequestEvent> {
```

Comment:
```suggestion
  export interface RequestIter extends AsyncIterable<RequestEvent> {
```
I would also rename it to `HttpConn` or similar, it has `.rid` and `.close()` so it should be named like a resource.

Reply:
done